src/DataManagerPlugin.cs

```diff
@@ -12,12 +12,11 @@ namespace Silksong.DataManager;
 [Bep.BepInDependency("org.silksong-modding.i18n")]
 public partial class DataManagerPlugin : Bep.BaseUnityPlugin
 {
-    // This property will never be accessed before Start executes.
+    // These properties will never be accessed before Start executes.
     internal static DataManagerPlugin Instance { get; private set; } = null!;
-
     internal static Bep.Logging.ManualLogSource InstanceLogger => Instance.Logger;
 
-    internal CG.Dictionary<string, IOnceSaveDataMod> onceSaveDataMods = new();
+    internal CG.List<ManagedMod> ManagedMods = new();
```

Comment:
Nit: as a collection expression enjoyer this could also be
```suggestion
    internal CG.List<ManagedMod> ManagedMods = [];
```